behat.yml

```diff
@@ -27,3 +26,10 @@ default:
         login_form_selector: '#user-login-form'
 imports:
   - behat.local.yml
+  - modules/lightning_features/lightning_core/tests/behat.yml
```

Comment:
I'm pretty sure this will break BLT integration. This introduces a new requirement for executing Lightning tests: importing Lightning-specific configuration via Behat. This is not performed automatically for us by the Drupal Extension, and so will need to be done manually for any project wishing to execute Lightning tests.